Files/Helpers/ExternalResourcesHelper.cs

```diff
@@ -25,6 +25,20 @@ public class ExternalResourcesHelper
 
         public string CurrentThemeResources { get; set; }
 
+        public Task LoadWindowsVersionStyle()
+        {
+            OperatingSystem osVersion = Environment.OSVersion;
+            int majorVersion = osVersion.Version.Major;
+
+            if (majorVersion == 10) // Windows 10 style
+            {
+                var windows10Style = new ResourceDictionary() { Source = new Uri("ms-appx:///ResourceDictionaries/Windows10Style.xaml") };
+                Application.Current.Resources.MergedDictionaries.Add(windows10Style);
+            }
+
+            return Task.CompletedTask;
+        }
```

Comment:
Should we load this theme no matter the Windows version?

Reply:
If you mean about Windows 10 version, I think there is no Windows 10 version that have the "after" screenshot instead the "before".

If you mean about Windows 11, this shouldn't be loaded to Windows 11, and I think that this way it won't.